src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,121 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIAL_CAPACITY = 10;
+    private static final int FIRST_ELEMENT_OF_ARRAY = 0;
+    private Object[] elements;
+    private int size;
+
+    public ArrayList() {
+        elements = new Object[INITIAL_CAPACITY];
+    }
+
     @Override
     public void add(T value) {
-
+        if (isFull()) {
```

Comment:
Move this logic to another method. Fix in all places
[See here](https://github.com/mate-academy/jv-array-list/pull/741#pullrequestreview-1223384788)